localization.py

```diff
@@ -16,33 +16,38 @@ def __init__(self, localizationType=rawSensor):
 
         super().__init__("localizer")
         
-        # TODO Part 3: Define the QoS profile variable based on whether you are using the simulation (Turtlebot 3 Burger) or the real robot (Turtlebot 4)
+        # Part 3: Define the QoS profile variable based on whether you are using the simulation (Turtlebot 3 Burger) or the real robot (Turtlebot 4)
         # Remember to define your QoS profile based on the information available in "ros2 topic info /odom --verbose" as explained in Tutorial 3
-
-        odom_qos=...
+        odom_qos=QoSProfile(reliability = 2, durability=2, history=1, depth=10)
```

Comment:
Will fix

Reply:
Fixed in [36c42f2](https://github.com/DiegoJohnson25/MTE544_group7/pull/1/commits/36c42f25f7c92cfbdc38570cae5b0d525fbab86e)